openslides/utils/autoupdate.py

```diff
@@ -95,9 +97,16 @@ def send_data(message):
             'action': 'deleted' if message['is_deleted'] else 'changed'}
         if not message['is_deleted']:
             data = access_permissions.get_restricted_data(full_data, user)
+
             if data is None:
-                # There are no data for the user so he can't see the object. Skip him.
-                continue
+                # send data if user can see the projector and data is projected
```

Comment:
This wont work because our current ProjectorRequirement approach relays on views and viewsets instead of models (root rest elements as we call them). Maybe we have to change these things here and in rest_api.py line 127.